desc/vmec.py

```diff
@@ -242,8 +248,9 @@ def save(cls, eq, path, surfs=128, verbose=1):  # noqa: C901 - FIXME - simplify
         NFP = eq.NFP
         M = eq.M
         N = eq.N
-        M_nyq = M + 4
-        N_nyq = N + 2 if N > 0 else 0
+        M_nyq = M + 4 if M_nyq is None else M_nyq
```

Comment:
are these still the correct defaults? It sounded like `N_nyq = NZETA*2`?

Reply:
also you can use `setdefault` for a cleaner version of this

Reply:
These are still correct, I think. At least VMEC by default gives M nyq of Mpol-1+4, and Mpol-1 = our Mpol